docs/how-to-add-a-validation-check.rst

```diff
@@ -1,3 +1,87 @@
 How to add a validation check
 =============================
 
+Data validation takes place in the `lib-cove-ocds <https://github.com/open-contracting/lib-cove-ocds>`_ library and are presented in the UI via templates in cove-ocds.
+
+'Common checks' process the various different objects in a release, so the data can be validated against the schema and statistics about different aspects of the data (eg. the number of contracts, the range of dates, or number of unique organisations present) can be presented in the UI. This happens in ``lib/common_checks.py``.
+
+'Additional checks' are for data validation beyond what is covered by validation against the `OCDS Schema <https://github.com/open-contracting/lib-cove-ocds/blob/master/libcoveocds/schema.py>`_ - that is, the results of these checks may suggest an issue with the data where the issue does not cause data to be invalid against the schema - and are implemented in ``lib/additional_checks.py``. An example of such a check is identifying when fields, objects and arrays exist but are empty or contain only whitespace.
+
+What follows is an outline of how to add a check. This will involve making changes in both `lib-cove-ocds` and `cove-ocds`.
```

Comment:
Is it meant to be double ticks not single ticks? I don't have time to compile and check before a meeting tho :-)

Reply:
yeah it works, I already compiled: https://ocds-data-review-tool.readthedocs.io/en/docs-howto/how-to-add-a-validation-check.html

Single ticks just does italic afaik

Reply:
Fair enough, I was confused by use of single and double ticks but now I check it looks good